test/selenium/lib/view/successful_page/accessors.js

```diff
@@ -12,7 +12,14 @@ function SuccessfulPageAccessor(driver) {
 
 SuccessfulPageAccessor.prototype = {
    get successMessageLocator() {
-        return this.driver.findElement(SELECTORS.successMessage);
+        var driver = this.driver;
```

Comment:
Old habits. I didn't realise Node 4 had support for arrows now. Since I'm using it now. PR has been updated to use arrows.